category/application/fake/category_service.go

```diff
@@ -0,0 +1,75 @@
+package fake
+
+import (
+	"context"
+
+	"flamingo.me/flamingo/v3/framework/flamingo"
+
+	"flamingo.me/flamingo-commerce/v3/category/domain"
+)
+
+//go:generate go run github.com/go-bindata/go-bindata/v3/go-bindata -nometadata -pkg fake -prefix mock/ mock/
+
+// CategoryService returns category test data
+type CategoryService struct {
+	testDataFiles map[string]string
+	logger        flamingo.Logger
+}
+
+var _ domain.CategoryService = new(CategoryService)
+
+// Inject dependencies
+func (f *CategoryService) Inject(logger flamingo.Logger, config *struct {
+	TestDataFolder string `inject:"config:commerce.category.fakeService.testDataFolder,optional"`
+}) {
+	f.logger = logger.WithField(flamingo.LogKeyModule, "category")
```

Comment:
Maybe provide the `flamingo.LogKeyCategory` with something like `fakeService` so that filtering logs later is easier.